Charlie4.rb

```diff
@@ -0,0 +1,40 @@
+mbta={
+red: ["Alewife", "Davis", "Porter", "Harvard", "Central", "Kendall/MIT", "Park Street","South Station"],
+green: [ "Haymarket", "Government Center", "Park Street", "Boylston", "Arlington", "Copley"],
+orange: ["North Station", "Haymarket2", "Park Street", "State Street", "Downtown Crossing", "Chinatown", "Tufts Medical Center"]
+}
+
+puts "Thank you for riding the MBTA!"
+puts "Where will you start your trip?"
+start = gets.chomp.downcase.to_sym
+
+# if (mbta[red].has_key?(start))
+#     puts "That's on the red line"
+#     #origin_line = red
+# elsif (mbta[green].has_key?(start))
+#      puts "That's on the green line"
+#     #origin_line = green
+# elsif (mbta[orange].has_key?(start))
+#      puts "That's on the orange line"
+#     #origin_line = orange
+# end
+
+puts 'Which line will you be riding?'
+start_line = gets.chomp.downcase.to_sym
+
+puts 'What is your destination stop?'
+destination = gets.chomp.downcase.to_sym
```

Comment:
Your destination shouldn't be a symbol. It's stored as a string inside an array.